### PR TITLE
WIP: Homepage

### PR DESCRIPTION
**Pull request by dzucconi**
Closes https://github.com/aredotna/ervell/issues/631

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,13 @@ The general idea here is something minimal, utilitarian, unobtrusive and adaptib
...
+
+-----
+
+## Deploy staging
+
+```
+make deploy-staging branch=branch
+```
+
+(This doesn't work.)
```
Comment (dzucconi) [inline README.md]: But it would be nice

**Comment by dzucconi** [comment]
Closes https://github.com/aredotna/ervell/issues/689

homepage

homepage-mobile

**Comment by dzucconi** [comment]
@broskoski https://github.com/aredotna/ervell/issues/689 checked off

**Comment by dzucconi** [comment]
@broskoski you're going to want to rebase and then pull request against this branch (lemme know if you need a git assist)

**Comment by dzucconi** [comment]
I started to rip out auth but it's really really spaghetti'd in there. I'm going to try to take a more granular approach. https://github.com/dzucconi/ervell/commit/4abddce694d4cab81e6156c9ec967391cde48fc0

**Comment by broskoski** [comment]
God bless you for this epic PR. I'm merging it.

**Pull request merged**